packages/grpc-native-core/src/common.js

```diff
@@ -112,6 +112,20 @@ exports.getMethodType = function(method_definition) {
   }
 };
 
+/**
+ * Given an array of property names and an array of values,
+ * combine the two into an object map.
+ * Equivalent to _.zipObject.
```

Comment:
This should be marked `@private` in the JSDoc so that it doesn't show up in the generated documentation, since it's not part of the public API.

Reply:
Good call!